lib/config.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+let path = require("path");
+
+module.exports = function readConfig(rootDir, filepath = "faucet.config.js") {
```

Comment:
* Again... why give it a name?
* Where is this function called? 😕

Reply:
* See above.
* Good catch... 🤦‍♂️ Fixed in e64a927a1263ef3d8cef68307d0ac7a91438e8b8. (We'll have to squash this, of course.)